internal.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+type Internal struct {
+	Bitswap InternalBitswap
+}
+
+const (
+	DefaultBitswapEngineBlockstoreWorkerCount = 2000
+	DefaultBitswapTaskWorkerCount             = 500
+	DefaultBitswapEngineTaskWorkerCount       = 500
+	DefaultBitswapMaxOutstandingBytesPerPeer  = 1 << 21
+)
+
+// InternalBitswap contains knobs for tuning bitswap resource utilization.
+// The knobs (below) document how their value should related to each other.
+// Whether their values should be raised or lowered should be determined
+// based on the metrics active_tasks, pending_tasks, pending_block_tasks and active_block_tasks
```

Comment:
The documentation is needed, but realistically nobody looks at code comments. Instead they're more likely to look at https://github.com/ipfs/go-ipfs/blob/master/docs/config.md (or for experimental features https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md).

If someone feels strongly about adding a new `internal-config.md` file we can do that otherwise let's just toss it in a section in `config.md`.